cmd/ovirt-populator/main.go

```diff
@@ -0,0 +1,262 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"forklift.konveyor.io/ovirt-populator/pkg/v1beta1"
+	populator_machinery "github.com/kubev2v/lib-volume-populator/populator-machinery"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/klog/v2"
+)
+
+const (
+	prefix     = "forklift.konveyor.io"
+	mountPath  = "/mnt/"
+	devicePath = "/dev/block"
+)
+
+var version = "unknown"
+
+const (
+	groupName  = "forklift.konveyor.io"
+	apiVersion = "v1beta1"
+	kind       = "OvirtVolumePopulator"
+	resource   = "ovirtvolumepopulators"
+)
+
+func main() {
+	var (
+		mode         string
+		engineUrl    string
+		secretName   string
+		diskID       string
+		fileName     string
+		crName       string
+		crNamespace  string
+		httpEndpoint string
+		metricsPath  string
+		masterURL    string
+		kubeconfig   string
+		imageName    string
+		showVersion  bool
+		namespace    string
+	)
```

Comment:
that's a bit unusual to see.. if most of the variables are of the same type and there's no assignment, why not a single or multiple line(s) of
```golang
var x,y,z string
var showVersion bool
```